OpenKh.Tools.ModsManager/ViewModels/SetupWizardViewModel.cs

```diff
@@ -172,81 +181,194 @@ public SetupWizardViewModel()
                 FileDialog.OnOpen(fileName => OpenKhGameEngineLocation = fileName, _openkhGeFilter));
             SelectPcsx2Command = new RelayCommand(_ =>
                 FileDialog.OnOpen(fileName => Pcsx2Location = fileName, _pcsx2Filter));
+            SelectPcReleaseCommand = new RelayCommand(_ =>
```

Comment:
In the mod loader / Panacea branch, this uses

```cs
SelectPcReleaseCommand = new RelayCommand(_ =>
                FileDialog.OnFolder(path => PcReleaseLocation = path));
```

which I think would be preferred to avoid confusion for users knowing which executable they need to choose. Should also make integrating the other branch easier if this is using the same approach as that one.